app/Models/ContributionPlan.php

```diff
@@ -132,4 +132,13 @@ public function getIsCompletedAttribute()
     {
         return $this->contributionStatus === self::STATUS_COMPLETED;
     }
+
+
+    public function getContributionInterest()
```

Comment:
Please rename p, i and t with proper variable names that explain what they are

Reply:
@omadoyeabraham 
variables renamed